third-party/snyk/snapshot.json

```diff
@@ -58,7 +58,7 @@
     ],
     "jquery":[
       {"id":"npm:jquery:20160529","severity":"low","semver":{"vulnerable":["=3.0.0-rc.1"]}},
-      {"id":"npm:jquery:20150627","severity":"medium","semver":{"vulnerable":["<3.0.0-beta1 >1.12.3 || <1.12.0 >=1.4.0"]}},
+      {"id":"npm:jquery:20150627","severity":"medium","semver":{"vulnerable":["<1.12.2",">=1.12.3 <2.2.2",">=2.2.3 <3.0.0"]}},
```

Comment:
I'm not a semver parsing expert, but our sample artifacts changed because our jquery `2.1.1` inclusion changed. Shouldn't it not have changed ?

**Before**
✅ 2.1.1 < 3.0.0-beta1
✅ 2.1.1 > 1.12.3

**After**
✅ 2.1.1 < 2.2.2
✅ 2.1.1 >= 1.12.3

🤔

Reply:
looks like we got a bug :/

![image](https://user-images.githubusercontent.com/39191/53902199-8c9abf80-3ff5-11e9-9f1d-6101338cd741.png)


i'll put up a fix